HW04.md

```diff
@@ -0,0 +1,380 @@
+Exploring Gun Deaths in America
+================
+
+## Objectives
+
+  - To explore gun death data curated by FiveThirtyEight [original data
+    available here](https://github.com/fivethirtyeight/guns-data) by
+      - Month
+      - Intent
+      - Age and sex
+      - Education
+      - Season
+      - Intent by race
+      - Police-relatedness
+
+## Read in the data
+
+The gun\_deaths.csv is provided in the repository.
+
+``` r
+library(tidyverse)
+library(ggplot2)
+# load tidyverse packages, including ggplot2
+
+# read in the data
+gun_deaths <- read.csv("gun_deaths.csv")
+
+head(gun_deaths)
+```
+
+    ##   year month  intent police sex age
+    ## 1 2012     1 Suicide      0   M  34
+    ## 2 2012     1 Suicide      0   F  21
+    ## 3 2012     1 Suicide      0   M  60
+    ## 4 2012     2 Suicide      0   M  64
+    ## 5 2012     2 Suicide      0   M  31
+    ## 6 2012     2 Suicide      0   M  17
+    ##                                                   race hispanic           place
+    ## 1                               Asian/Pacific Islander      100            Home
+    ## 2                                                White      100          Street
+    ## 3                                                White      100 Other specified
+    ## 4                                                White      100            Home
+    ## 5                                                White      100 Other specified
+    ## 6 Not Availabletive American/Not Availabletive Alaskan      100            Home
+    ##      education
+    ## 1          BA+
+    ## 2 Some college
+    ## 3          BA+
+    ## 4          BA+
+    ## 5       HS/GED
+    ## 6 Less than HS
+
+## Generating a data frame that summarizes the number of gun deaths per month and printing using `kable()`.
+
+Knitr is part of the knitr package. Allows you to change column names
+and add captions and make pretty tables in your knitted document. Hint:
+set the argument format = “markdown”
+
+``` r
+deaths_per_month <- gun_deaths %>%
+    count(month)%>%
+    rename(deaths = n)
+
+deaths_per_month<-transform(deaths_per_month, MonthAbb = month.abb[month])
+deaths_per_month$MonthAbb <- factor(deaths_per_month$MonthAbb, levels = c("Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"))
```

Comment:
There is actually a `month.abb` constant in R for these abbreviations and that's very interesting to know. By the way, in line 65, I assume the vector after "levels =" can be substituted by this `month.abb` constant, which might increase readability.